core/src/main/java/jenkins/slaves/JnlpSlaveAgentProtocol4.java

```diff
@@ -175,11 +149,18 @@ public String getDisplayName() {
 
     @Override
     public String getName() {
-        return handler.getName();
+        return "JNLP4-connect"; // matches JnlpProtocol4Handler.getName
```

Comment:
Extract to constant?

Reply:
Could, though this would require a new `remoting` release.

Reply:
Ah right, missed the fact it was somewhere else.